stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 8.1.0
+version: 8.1.1
```

Comment:
side note: I been wondering if/where best to keep changelog (bugfixes, features, updates, breaking changes, docs, ...) 

something that possibly [clog](https://github.com/clog-tool/clog-cli) could help with 
- https://github.com/spinnaker/spinnaker
- https://github.com/angular/angular
seem. to use

Reply:
I'd love to have a changelog section on the Readme.md

Feel free to add in your next PRs